### DISCUSSION TITLE
Gopro Hero 13 Black with Labs Firmware doesn't work with Quick?

### DISCUSSION THREAD

**Discussion by AirKhann** [gopro/labs#1086] [General]
![image](https://github.com/user-attachments/assets/9d0ba075-16ea-4de7-b1c6-2b4ed9612f4d)

Since i've installed the labs firmware to enable NR01=1 and BITR=200, i can't control my Gopro 13 over Quick anymore, changing profiles also doesnt work. And somehow the settings control also doesnt work just the media download and live view and the on off function works and sometimes i get a wifi cant connect message but then it connects anyway.

**Comment by dnewman-gpsw** [comment]
No idea why that might be, those controls should not affect Quik.  New Labs release out Tuesday, try that.

**Reply by toroe0815** [reply]
Same problem with my GoPro 13 Black. I will try it tomorrow.

**Reply by toroe0815** [reply]
Just tested. Works now with fw 1.12.70.

**Reply by dnewman-gpsw** [reply]
The official 1.12.70 is tomorrow.  You will want that.

**Reply by AirKhann** [reply]
Okay, i installed v01.12.00 and then everything worked fine in Quik but after i installed the new Labs Firmware for the Hero 13 (v01.12.70) i got the same problem as before, i also did a reset in Labs and in the settings menu (factory reset?) all my profiles where deleted but Quik still doesn't work for me with the Labs Software any idea or suggestions? If i go back to v01.12.00 it works but as soon as i install labs it doesn't work anymore no matter if i have active gopro labs settings or not.

**Reply by dnewman-gpsw** [reply]
Can you film what is not working?  As I can't get it to fail.

![image](https://github.com/user-attachments/assets/f921fe18-6eec-4a8c-af07-6afe870bbc6f)

That ```null``` you are getting, suggests a preset corruption.  On Labs scan !PRESET, follow the on screen instructions until "approved" is displayed.  Also on Quik, delete the camera, so you are starting refresh.  Labs does not alter Quik connections in any way, but it does mess with presets (in 1.10.70 not correctly), and a messed-up preset is not automatically erased with a firmware update.

**Reply by AirKhann** [reply]
Okay i deleted the camera from Quik i also did scan !PRESET after approved i did also scan !FRESET after that i connected the camera to Quik and my Router and it still doesnt work with labs! Only on off and changing the camera between pro and easy control does work, record button does not work and switching profiles also doesnt work. This time i did not change any setting in the camera i also did not create a custom preset.

![image](https://github.com/user-attachments/assets/e163a972-721e-4bf1-969e-879eb088a460)

![image](https://github.com/user-attachments/assets/5ec8da09-c53b-40e4-a0b3-77ff31a8b182)

![image](https://github.com/user-attachments/assets/3a542a08-1d36-44b5-8895-5e74e40004de)

**Reply by AirKhann** [reply]
Going back to the normal Firmware (v01.12.00) does solve the problem. And everything works fine, but i need the labs Software its essential for me.

![image](https://github.com/user-attachments/assets/11647340-f24f-4503-a130-a4c0192508ac)
![image](https://github.com/user-attachments/assets/f40cca40-03c2-4052-bc83-854ffb169113)

**Reply by dnewman-gpsw** [reply]
I even switched my camera language to match yours, no issues here.  I'm not hearing any other reports of this, so I do not have any leads on how to reproduce this behavior.  I think this might be iPhone, which I don't have, so I can't replicate everything.

**Reply by AirKhann** [reply]
I did that too, i switched the camera to english but didnt help i created my presets again with the normal Firmware, i even changed the names of the preset with the normal Firmware (v01.12.00) in Quik, it was no problem.

![image](https://github.com/user-attachments/assets/90e449ff-c8ad-464d-b7e4-41d748b0bb88)

But with gopro labs installed again (v01.12.70) the controls and presets disappeared again in Quik

![image](https://github.com/user-attachments/assets/af959012-96ac-4bfb-b2c7-818bc534e9eb)
![image](https://github.com/user-attachments/assets/11d74c3c-420d-4cb0-b04c-7ef274dbbb16)

**Comment by dnewman-gpsw** [comment]
Labs can be disabled by adding a file ```no_labs.txt``` within the MISC folder on the SD card. This will tell me if the error relates to Labs just being installed, or if the Labs processing on the camera is somehow interfering with Quik.  Add the file to the SD Card, start-up the camera; you will get no Labs messaging during the boot.  Does Quik work in this state?  You can then remove the file to restore Labs functions.

**Reply by AirKhann** [reply]
I did what you said, and there was no Labs message during boot, but the results are the same i cant record or change presets over Quik. I will try my Android Tablet maybe that will work. I also reinstalled Quik on my iPhone 12 Pro Max but that didnt help either. Changing the language on my phone to english (us) didnt help changing the camera language to english also didnt help.

![image](https://github.com/user-attachments/assets/29c35561-8084-4ef8-99b7-1353fc9ab8fe)
![image](https://github.com/user-attachments/assets/e5ab5d0e-6c4d-4783-a187-18352881f5b0)

**Reply by dnewman-gpsw** [reply]
Redownload and reinstall labs, from the updated link: https://bit.ly/LABS_H13_1_12_70

If labs wasn't running, they something wrong with either the FW install on the camera (weird) or the Quik (latest version?) on the phone (also weird).  Nothing is it very logically here.

**Reply by AirKhann** [reply]
I don't know what kind of magic you did to the file, but installing it did solve the problem, it now works. Thanks!

PS: Yes, Quik is on the latest version, i guess, because there are no updates in the AppStore. My version is 13.1 (15635)

![image](https://github.com/user-attachments/assets/b0d4b7e0-8167-4031-9611-cf5ce77971e1)

**Comment by dnewman-gpsw** [comment]
I have no idea.  Very happy to hear it is now working.

**Comment by Riebli** [comment]
I have the exact same problem as described above.
I immediately loaded the labs software onto a new Hero 13 and have the problem.
My solution: install the labs software 2 times one after the other and now it works: Strange but it works.

**Comment by Riebli** [comment]
![IMG_3365](https://github.com/user-attachments/assets/e13bf453-1599-401a-a31f-b116023513c8)
![IMG_3364](https://github.com/user-attachments/assets/2343786a-e3f3-48b0-9f09-dc5878c6e31e)
Unfortunately, a few settings are not possible, see pictures

**Reply by dnewman-gpsw** [reply]
Reset your Presets.  They are setting in Labs that can be displayed in Quik, as Labs is extending the camera, but default presets should show any undisplayable settings.

**Reply by Riebli** [reply]
I set the camera back to factory settings but again the same.
Got it with ios 18.0.1 app version 13.2 (15786)
Attempted

**Reply by dnewman-gpsw** [reply]
Doesn't do that for me. As 4:3 is rare on H13, what resolution and frame rate is the camera set to?

**Reply by Riebli** [reply]
Back to 01.12 now Ales is displayed and works

**Reply by Riebli** [reply]
![IMG_3366](https://github.com/user-attachments/assets/83d73845-1571-4aba-8ac8-1d58d2fcffef)

**Reply by Riebli** [reply]
It seems that the data from the camera is not read out correctly to me softtrue 01.12.70

Nothing has been changed in the settings of the two pictures

**Reply by Riebli** [reply]
I reinstalled the software 01.12.70 against the same problem 
Software with no_labs.txt turned off still the same thing on the camera has not been changed all data corresponds to the last picture

**Reply by dnewman-gpsw** [reply]
There is no Labs code involved with Quik, which is also likely why no_labs.txt makes no difference.  I'm not seeing it in the current work in progress to 1.20.70, so hopefully this will be address in the next release.

**Reply by Riebli** [reply]
ok Thanks for the efforts

**Reply by Riebli** [reply]
Hello 
I was able to solve the problem as following 
1 labs 01.12.70 installed 
2 https://gopro.github.io/labs/control/set/?cmd=!PRESET pressed 3x record button as described on the screen (fast)
3 Installed labs 01.12.70 again 
4 now everything is good in the Quik app

**Reply by AirKhann** [reply]
I have still issues with the App even after the 2 times in a row reinstall i can now record but some settings are listed as "unknown". Now i am on v1.20 and everything works fine, also in the app i will try it with labs when its released.

**Comment by AirKhann** [comment]
V1.20.70 works like V1.20 seemlessly with the Quik App everything works now as intended with the Quik App!

**Comment by airhockeyteam** [comment]
I'm **very** new to GoPro--Hero 13 Black is my first--but not new to photography or technology. I'm still figuring out Labs, but the QR code functionality appealed to me for using it in activities with the waterproof housing. I had the same issue described in this thread where a lot of the camera's functionality wouldn't work with the Quik app after installing the v01.12.70 Labs firmware. Went back to non-Labs v01.12 firmware, the v01.20, and the Quik issue was resolved. I just installed Labs v01.20.70, and like @AirKhann , it appears that everything with both Quik and Labs is working as intended. Thank you!!!

**Comment by mnott** [comment]
OK Gopro Hero here with Labs 01.30.72. I didn't have a connection to the Gopro and stupidly did not notice that installing labs had just disabled wireless connections (so I needed to switch them back on in the Gopro).
